app/templates/layouts/asylum-base.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="charset" content="utf-8">
```

Comment:
arbitrary. that's just how flask base sets things up (see the standard `templates/partials/_head.html` that's in the header of all other flask base apps. I move to keep this the way it is to maintain consistency with any other views we write (which'll probably use the `base.html` template instead of `asylum-base.html`